abis_mapping/types/metadata.py

```diff
@@ -23,6 +23,7 @@ class TemplateMetadata(pydantic.BaseModel):
     sampling_type: str
     template_url: str
     schema_url: str
+    template_status: str
```

Comment:
I'm trying to find a better name, like 'lifecycle_status` or `support_status`.
Also I think the type should be an `Enum`, with the values:
* "Beta"
* "Current"
* "Deprecated"

Reply:
I opt for `lifecycle_status`

Reply:
The phrase `lifecycle` is going to get confusing given we're already talking about biology?